<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="actionre_Start"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_eXit"/>
   </widget>
   <widget class="QMenu" name="menu_Solver">
    <property name="title">
     <string>&amp;Solver</string>
    </property>
    <addaction name="action_NextHint"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menu_Solver"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_NextHint"/>
   <addaction name="separator"/>
   <addaction name="actionre_Start"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_eXit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/reorder_black.png</normaloff>:/MainWindow/Resources/reorder_black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_eXit">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/exit_black.png</normaloff>:/MainWindow/Resources/exit_black.png</iconset>
   </property>
   <property name="text">
    <string>e&amp;Xit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/undo_black.png</normaloff>:/MainWindow/Resources/undo_black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/redo_black.png</normaloff>:/MainWindow/Resources/redo_black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_NextHint">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/question.png</normaloff>:/MainWindow/Resources/question.png</iconset>
   </property>
   <property name="text">
    <string>&amp;NextHint</string>
   </property>
  </action>
  <action name="actionre_Start">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/renew_black.png</normaloff>:/MainWindow/Resources/renew_black.png</iconset>
   </property>
   <property name="text">
    <string>re&amp;Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
